### PR TITLE
Load b64 images

### PR DESCRIPTION
**Pull request by JohanDjarvKarltorp**
closes issues #94 #93
- Fixes database performance
- User can now upload own icon when creating a new category
- changed branch connection icon to a square instead
- Does not longer save unnecessarily data (popup content, product image, icon image) to database

### CODE REVIEW

**Review by filipahlgren — APPROVED** [review]
good

File: js/map/loadLeafletMap.js [hunk 1]
```diff
@@ -393,157 +369,145 @@ let deleteOnClick = () => {
...
+            // Call addMarker function in add.js
+            map.on("click", add.marker);
+            document.getElementById("myMap").style.cursor = "pointer";
+        });
+    }
+};
 
 /**
- * loadProducts - Loads all products from database and with each category creates a new accordion
+ * loadClickEvent - Description
```
Comment (SpaceLenore) [inline js/map/loadLeafletMap.js]: Please describe this description

File: js/map/loadLeafletMap.js [hunk 2]
```diff
@@ -393,157 +369,145 @@ let deleteOnClick = () => {
...
  *
- * @returns {void}
+ * @returns {type} Description
```
Comment (SpaceLenore) [inline js/map/loadLeafletMap.js]: Please describe this description that is being returned

File: js/map/loadLeafletMap.js [hunk 3]
```diff
@@ -393,157 +369,145 @@ let deleteOnClick = () => {
...
+                    }
+
+                    accordions();
+                    show.activeObj();
+
+                    loadClickEvent();
+                    addPipeOnClick();
+                    addHouseOnClick();
+                } else {
+                    if (json.info == "token failed to validate") {
```
Comment (SpaceLenore) [inline js/map/loadLeafletMap.js]: Probably better to check the json.error parameter as it will always be true or false while the error message may vary.

**Pull request merged**